### PR TITLE
Staff Application Command Stage 1 Implemented

### PR DESCRIPTION
**Pull request by MountainAero**
Staff application command checks account age and membership time of the invoking user and sends the user the application link if they meet the requirements.

### CODE REVIEW

**Comment by Arborym** [comment]
Going to work on rebasing and prettifying the embeds. Will show account information using `fields` instead of `description`

File: cogs/Staffrequirements.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    @commands.slash_command(
+        name="check-requirements",
+        description="Check if the account meets requirements for staff.",
+    )
+    # @option(
+    #    name="user ID",
+    #    description="The uID of the account to check.",
+    #    #autocomplete=get_keywords,
+    # )
+    async def add_alert(self, ctx: commands.Context) -> None:
```
Comment (sebastiaan-daniels) [inline cogs/Staffrequirements.py:30]: Copy pasted  function name from other cog?

File: cogs/Staffrequirements.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    # Allows the user to set the keeptime to a value other than the default
+    @commands.slash_command(
+        name="check-requirements",
+        description="Check if the account meets requirements for staff.",
+    )
+    # @option(
+    #    name="user ID",
+    #    description="The uID of the account to check.",
+    #    #autocomplete=get_keywords,
+    # )
```
Comment (sebastiaan-daniels) [inline cogs/Staffrequirements.py:29]: should be removed if commented out

**Review by sebastiaan-daniels — DISMISSED** [review]

File: cogs/Staffrequirements.py (hunk 2 above)
Comment (Arborym) [reply cogs/Staffrequirements.py]: Rebased in https://github.com/Arborym/Pax-Academia/pull/58/commits/2700433aa6c2ddfa39d5dfbb320bb23a620ff985

File: cogs/Staffrequirements.py (hunk 1 above)
Comment (Arborym) [reply cogs/Staffrequirements.py]: Rebased in https://github.com/Arborym/Pax-Academia/pull/58/commits/2700433aa6c2ddfa39d5dfbb320bb23a620ff985

**Review by Arborym — APPROVED** [review]

**Pull request merged**